yaclt/yaclt
# Added dry-run option to prepare-release command

Resolves: #128 

## How to Test

1. Run ```yactl new -m "<your message here>"```
2. Run the command ```yactl prepare-release --dry-run``` from the master branch of a git project
3. Verify that CHANGELOG.md is unchanged and that the change preview contains the earlier added change
4. Run ```yactl prepare-release --dry-run --quiet``` and verify a contradiction error is printed

## Potential Side Effects/Things to Look Out For

Kebab-case option differs from the rest of the camel-cased options which required a slight workaround in the validate-argv-middleware.
